src/quo2/components/avatars/account_avatar/style.cljs

```diff
@@ -6,16 +6,20 @@
 (def default-padding 16)
 (def default-emoji-size 36)
 
+(def ^:private container-size
+  {:size-64 64})
+
 (defn get-border-radius
   [size]
   (case size
-    80 16
-    48 12
-    32 10
-    28 8
-    24 8
-    20 6
-    16 4
+    80       16
+    :size-64 16
```

Comment:
@clauxx - sizes is another part of the quo2 refactors that will need to be adjusted. We decided to start using the keyword `:size-<number>` however these uses have to be adjusted for the previous sizes which also means updating this across the uses in the codebase.

Reply:
👍 will add an issue which will track adjusting the size across quo2.